tests/config/serverFixtures.ts

```diff
@@ -49,9 +49,11 @@ export const serverFixtures: Fixtures<ServerFixtures, ServerWorkerOptions> = {
 
     const socksServer = socks.createServer((info, accept, deny) => {
       const socket = accept(true);
-      if (socket) {
-        // Catch and ignore ECONNRESET errors.
-        socket.on('error', () => {});
+      if (!socket)
+        return;
+      socket.on('data', data => {
+        if (!data.toString().startsWith('GET /'))
```

Comment:
Does it have to be 'GET' ? Can we wait for complete http request header instead?